broker/l2tp_broker.cfg.example

```diff
@@ -12,6 +12,10 @@ tunnel_id_base=100
 ; Reject connections if there are less than N seconds since the last connection.
 ; Can be less than a second (e.g., 0.1).
 connection_rate_limit=10
+; Runs broker.tunnel-connection-threshold if a IP address connect more than X times in N seconds
+connection_rate_limit_per_ip=30
+connection_rate_limit_per_ip_timescale=60
```

Comment:
```suggestion
connection_rate_limit_per_ip_count=30
connection_rate_limit_per_ip_times=60
```
(or similar)
Then you can write the docs as "more than COUNT times in TIME seconds", making it more clear than currently what X and N are.